.gitignore

```diff
@@ -1,18 +1,3 @@
 .vim/
 tmp/
-ipinfo/dist/
-grepip/dist/
-prips/dist/
-cidr2range/dist/
-cidr2ip/dist/
-range2cidr/dist/
-range2ip/dist/
-randip/dist/
-!ipinfo/dist/DEBIAN/
-!grepip/dist/DEBIAN/
-!prips/dist/DEBIAN/
-!cidr2range/dist/DEBIAN/
-!cidr2ip/dist/DEBIAN/
-!range2cidr/dist/DEBIAN/
-!range2ip/dist/DEBIAN/
-!randip/dist/DEBIAN/
```

Comment:
Ok sorry, I should've said this earlier but - we actually want to keep these. These files help us produce individual `.deb` files for each binary which we include in each binary's release. This is separate from the PPA stuff. Users should be able to choose between using the PPA vs. using individual `.deb` files.